template/src/store/createStore.js

```diff
@@ -0,0 +1,17 @@
+import { createStore, compose } from 'redux'
+import rootReducer from './reducers'
+
+function configureStore(initialState, history) {
+  
+  const composeEnhancers = typeof window === 'object' &&
+  window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ ?
```

Comment:
Hmm interesting, I haven't seen usage of `compose` from the window. Do we need this if we aren't adding any enhancers? I guess it is good to have in place for folks that want to use it, but maybe we should just use `compose` directly since it is already imported anyway?

Reply:
Yup, got it. What I can do is add the dev-tool enhancer using native `compose`. I am also adding an empty array for `middlewares`, in case the user wants to add it later, for convenience.